File: Coding Challenge 18/helloname_lovely.m
function helloname_lovely
    name = input('Hello, what is your name?\n','s');

    fprintf('Hello, %s.\n',name);

    if (strcmpi(name,'Max'))
        fprintf('What a lovely name!\n')
    elseif (strcmpi(name,'Taylor'))
        fprintf('What a lovely name!\n')
    end
end

File: Coding Challenge 18/helloname_lovely.rb
def helloname_lovely
    puts 'Hello, what is your name?'
    name = gets.chomp
    puts 'Hello, ' + name + '.'
    
    if name == 'Max'
        puts 'What a lovely name!'
    elsif name == 'Taylor'
        puts 'What a lovely name!'
    end
end